openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -38,9 +41,9 @@ def get_account_settings(requesting_user, username=None, configuration=None, vie
         based on who `requesting_user` is and the privacy settings of the user associated with `username`.
```

Comment:
Update all the places is the doc where "requesting_user" is referenced (there are quite a few).